content/en/docs/tasks/administer-cluster/manage-resources/cpu-constraint-namespace.md

```diff
@@ -20,7 +20,7 @@ in the namespace.
 
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
-Each node in your cluster must have at least 1 CPU.
+Your cluster must have at least 1 CPU available for use to run the task examples.
```

Comment:
There is text below:
> At this point, your Container might be running or it might not be running. Recall that a prerequisite
for this task is that your Nodes have at least 1 CPU. If each of your Nodes has only
1 CPU, then there might not be enough allocatable CPU on any Node to accommodate a request
of 800 millicpu. If you happen to be using Nodes with 2 CPU, then you probably have
enough CPU to accommodate the 800 millicpu request.

Does that need to change as well?

Reply:
I missed that one! Fix is on the way!